alerts/google-network-intelligence-center/packet-loss-high.v1.json

```diff
@@ -8,18 +8,22 @@
   "conditions": [
     {
       "displayName": "Packet Loss higher than 5%",
-      "conditionMonitoringQueryLanguage": {
-        "query": "fetch gce_zone_network_health :: networking.googleapis.com/cloud_netslo/active_probing/probe_count\n | {\n   filter metric.result = 'failure'\n   | group_by [resource.region, metric.remote_region], 60s, .sum\n ;\n   group_by [\n     resource.region,\n     metric.remote_region\n   ], 60s, .sum\n   | filter val() >= 1 '1'\n }\n | ratio | condition val() > 0.05",
+      "conditionPrometheusQueryLanguage": {
         "duration": "300s",
-        "trigger": {
-          "count": 1
-        }
+        "evaluationInterval": "60s",
+        "query": "(\n  (sum by (region, remote_region) (\n    {\"networking.googleapis.com/cloud_netslo/active_probing/probe_count\", monitored_resource=\"gce_zone_network_health\", result=\"failure\"}\n  ))\n  /\n  clamp_min(\n    sum by (region, remote_region)(\n      {\"networking.googleapis.com/cloud_netslo/active_probing/probe_count\", monitored_resource=\"gce_zone_network_health\"}\n    ), 1\n  ) \n) > 0.05"
```

Comment:
While this probably will work because the evaluation interval and the point interval match up (and it's a ratio), networking.googleapis.com/cloud_netslo/active_probing/probe_count is a delta and therefore it's more correct (and more performant in monarch) to wrap both sides in a rate([60s])